src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -103,24 +103,23 @@ class CrossDeviceLinkUI extends Component {
       copySuccess: false,
       sending: false,
       error: {},
-      validNumber: true,
+      validNumber: true
     }
   }
 
   linkId = `${process.env.BASE_32_VERSION}${this.props.roomId}`
 
   linkCopiedTimeoutId = null
 
-  sendLinkClickTimeoutId = null
-
-  copyToClipboard = (e) => {
-    this.linkText.select()
-    document.execCommand('copy')
-    e.target.focus()
+  onCopySuccess() {
```

Comment:
minor: if you turn this into an arrow function you won't have to bind the `this`
We tend to do this for method which are meant to be passed as callbacks (usually methods called `on...`)

Reply:
Have made the change